scripts/deploy/nlp/deploy_triton.py

```diff
@@ -178,6 +180,19 @@ def get_args(argv):
         choices=['TensorRT-LLM', 'vLLM', 'In-Framework'],
         help="Different options to deploy nemo model.",
     )
+    parser.add_argument(
+        "-srs",
+        "--start_rest_service",
+        default="False",
+        type=str,
+        help="Starts the REST service for OpenAI API support",
+    )
+    parser.add_argument(
+        "-sha", "--service_http_address", default="0.0.0.0", type=str, help="HTTP address for the REST Service"
+    )
+    parser.add_argument(
+        "-sp", "--service_port", default=8000, type=int, help="Port for the Triton server to listen for requests"
```

Comment:
Can we change the default to something else? I guess this is the default port for triton.

Reply:
Fixed it in the latest commit.